### ISSUE TITLE
EssentialsGeoIP ''null''

### ISSUE THREAD

**Issue by Sikatsu** [EssentialsX/Essentials#1799]
Since updating EssentialsGeoIP from jenkins, everyone seems to come out of ''null''. I updated my config.

https://pastebin.com/0cX7ATGB

**Comment by Sikatsu** [comment]
Enabling cities within the config fixes this issue. But if I don't want it to show cities it seems to be bugged.

**Comment by k-jiang** [comment]
Sorry for the bug. Try delete the Country.mmdb files and restart your server. FYI the size of Country.mmdb should be at least 3 mb.
Fix should be available soon. #1784 

Let me know if you meet more GeoIP bugs. I will fix them.

**Comment by Sikatsu** [comment]
Players still come out of ''null'' if I disable cities.

http://prntscr.com/i4u9ic

The file is 3,3mb.

http://prntscr.com/i4u9ph

I made sure to delete it and restart the server.

**Comment by mdcfe** [comment]
Do any errors appear in the console when a player joins the game?

**Comment by Sikatsu** [comment]
There are no errors.

**Comment by k-jiang** [comment]
@Sikatsu That might be the problem of GeoIP2 database itself. Or the problem with your locale setting in Essentials' config.yml. Can you tell me what's your Essentials' `locale` setting (NOT EssentialsGeoIP)?
The setting looks like this:
```
# Set the locale for all messages.
# If you don't set this, the default locale of the server will be used.
# For example, to set language to English, set locale to en, to use the file "messages_en.properties".
# Don't forget to remove the # in front of the line.
# For more information, visit http://wiki.ess3.net/wiki/Locale
locale: en
```

And, could you pm me the players IP addresses whom encounter with the problem, or just their subnet address ([search the IP here](https://bgp.he.net/)) if concern about privacy. I will test the subnet with the MaxMind's Country database.

**Comment by Sikatsu** [comment]
It's set to ''nl'' and I use a custom language file for it.
It happens with all IP adresses btw.

**Comment by k-jiang** [comment]
Problem found. It is due to missing locale support from the database.

It seems that I forgot to check locale in Country queries. I will create a new PR to fix this once @SupaHam approved #1784 . It should be done soon.

For now you can enable `show-cities` just for a while. Or if you really need "Country only" feature, you can temporary change your Essentials' locale to `en` until the fix is released. Sorry for the inconveniences.

**Comment by Sikatsu** [comment]
I'll wait for the fix in that case. Thanks for your reply!
